config/routes.rb

```diff
@@ -16,4 +16,6 @@
 
   draw :placements
   draw :claims
+
+  get :healthcheck, controller: :heartbeat
```

Comment:
Logically, this is all good 👍 Out of curiousity, why is the controller "Heartbeat" and the action "healthcheck"? Are you planning to add additional heartbeats?

Reply:
I don't like having methods that are named the same as the class they are in, and this way we can add additional heartbeats if we need to